specs/withdrawals.md

```diff
@@ -175,7 +182,7 @@ The following inputs are required to prove and finalize a withdrawal:
 These inputs must satisfy the following conditions:
 
 1. The `l2BlockNumber` must be the block number that corresponds to the `OutputProposal` being proven.
-1. `L2OutputOracle.getL2OutputAfter(l2BlockNumber)` returns a non-zero `OutputProposal`.
+1. `L2OutputOracle.getL2Output(l2BlockNumber)` returns a non-zero `OutputProposal`.
```

Comment:
The `getL2Output` function takes the `l2OutputIndex` not `l2BlockNumber`.

See [L2OutputOracle.sol#L239-L245](https://github.com/ethereum-optimism/optimism/blob/qbzzt/230410-specs-withdrawals/packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol#L239-L245):
```solidity
function getL2Output(uint256 _l2OutputIndex)
    external
    view
    returns (Types.OutputProposal memory)
{
    return l2Outputs[_l2OutputIndex];
}
```

Reply:
Thank you, fixed (it also required a fix a few lines higher)